components/dashboard/src/settings/Teams.tsx

```diff
@@ -0,0 +1,767 @@
+                        <div className="px-4 self-center w-1/12">
+                            <div className={"rounded-full w-3 h-3 text-sm align-middle bg-gitpod-kumquat"}>
+                                &nbsp;
+                        </div>
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-5/12">
+                            <span className="my-auto font-medium truncate overflow-ellipsis">{pendingPurchase.plan.name}</span>
+                            <span className="text-sm my-auto text-gray-400 truncate overflow-ellipsis">Purchased on {formatDate(new Date().toString())}</span>
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-5/12">
+                            <span className="my-auto truncate text-gray-500 overflow-ellipsis">{"–"}</span>
+                            <span className="text-sm my-auto text-gray-400">Payment in Progress</span>
+                        </div>
+                        <div className="my-auto flex w-1/12 pl-8">
+                            <div className="self-center hover:bg-gray-200 rounded-md cursor-pointer w-8">
+                                <ContextMenu menuEntries={[]}>
+                                    <img className="w-8 h-8 p-1" src={ThreeDots} alt="Actions" />
+                                </ContextMenu>
+                            </div>
+                        </div>
+                    </div>
+                )}
+                {getActiveSubs().map((sub, index) => (
+                    <div key={"team-sub-" + sub.id} className="flex-grow flex flex-row hover:bg-gray-100 rounded-xl h-16 w-full">
+                        <div className="px-4 self-center w-1/12">
+                            <div className={"rounded-full w-3 h-3 text-sm align-middle bg-green-500"}>
+                                &nbsp;
+                        </div>
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-3/12">
+                            <span className="my-auto font-medium truncate overflow-ellipsis">{getPlan(sub)?.name}</span>
+                            <span className="text-sm my-auto text-gray-400 truncate overflow-ellipsis">Purchased on {formatDate(sub?.startDate)}</span>
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-2/12">
+                            <span className="my-auto truncate text-gray-500 overflow-ellipsis">{slots.filter(s => s.teamSubscription.id === sub.id).length || "–"}</span>
+                            <span className="text-sm my-auto text-gray-400">Members</span>
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-5/12">
+                            <span className="my-auto truncate text-gray-500 overflow-ellipsis">{"–"}</span>
```

Comment:
**question**: This is empty for now, right?

Reply:
I haven't found the source for that info, yet.